From: Carlos Maddela <e7appew@gmail.com>
Date: Thu, 22 Dec 2016 11:19:20 +1100
Description: Fix all warnings except for those related to deprecated API.
Bug-Debian: https://bugs.debian.org/748416
Forwarded: no
Last-Update: 2017-01-05
---
diff --git a/scsi-layer.c b/scsi-layer.c
index 64a3dba..3be21ff 100644
--- a/scsi-layer.c
+++ b/scsi-layer.c
@@ -31,7 +31,9 @@
 
 static int query_type(DeviceHandle*, int);
 static gint64 query_size(Image*);
+#if 0
 static int query_copyright(DeviceHandle*);
+#endif
 
 static int read_dvd_sector(DeviceHandle*, unsigned char*, int, int);
 static int read_cd_sector(DeviceHandle*, unsigned char*, int, int);
@@ -1649,6 +1651,7 @@ reset_mode_page:
  * Find out whether we are allowed to create an image from the DVD.
  */
 
+#if 0
 static int query_copyright(DeviceHandle *dh)
 {  Sense sense;
    AlignedBuffer *ab = CreateAlignedBuffer(2048);
@@ -1709,6 +1712,7 @@ static int query_copyright(DeviceHandle *dh)
 
    return result;
 }
+#endif
 
 /*
  * See whether a sector lies within the user area.
